web/pages/leagues.tsx

```diff
@@ -191,16 +289,29 @@ const CohortTable = (props: {
                   isUser={currUserId === user.id}
                 />
               )}
-
-              {promotionCount > 0 && i + 1 === promotionCount && (
+              {doublePromotionCount > 0 && i + 1 === doublePromotionCount && (
```

Comment:
Shouldn't this line also have a tooltip explainer? I feel it's confusing right now.